dartagnan/src/main/java/com/dat3m/dartagnan/wmm/Wmm.java

```diff
@@ -41,6 +41,8 @@ public List<Axiom> getAxioms() {
         return axioms;
     }
 
+    public Set<Relation> getRelations() { return relationRepository.getRelations(); }
```

Comment:
`Wmm.getRelationRepository().getRelation()` is also used all over the place. Shall we also provide a convenient method `Wmm.getRelation()` (I know @ThomasHaas wants to completely rewrite how we handle Wmms so this could be part of that intended change)

Reply:
I agree with this change (I was contemplating about doing it already), but I think we can defer it to the rework of `Wmm`.
That rework definitely intends to get rid of this ugly `RelationRepository` business.